client/locations-list-map.js

```diff
@@ -349,6 +350,15 @@ function createRequesterMarkerContent(entry, separator) {
     );
   }
 
+  if (timestamp) {
+    const date = new Date(timestamp);
+    const options = { year: 'numeric', month: 'long', day: 'numeric' };
+    contentTags.push(
+      ce('div', 'label', ctn($.i18n('ftm-date-added'))),
+      ce('div', 'value', ctn(date.toLocaleDateString(undefined, options)))
```

Comment:
is `undefined` here defaulting it to the browser's locale? I feel like since we have a bunch of logic dealing with locale, we should use the calculated locale here to ensure that there aren't weird edge cases

Reply:
Yes, it's defaulting to browser. I don't know how to pull that in, how do I get the user's locale? I need the country and language so it's a full locale like: "de-DE"

Reply:
country code is accessible as `gCountryCode` -- https://github.com/findthemasks/findthemasks/blob/master/client/locations-list-map.js#L22

I'm not 100% sure how to get `locale` in this module...you could add `getCurrentLocaleParam` to the i18nutils import (`import { getMapsLanguageRegion } from './i18nUtils.js';`) and then just use that function. Surprising that we don't have it already being used in `locations-list-map` anywhere but I guess we've gotten away without it

Reply:
I don't think gCountryCode is correct. I just checked and we are using "uk" for the United Kingdom when it's ISO code is "GB" I think (unless UK works on browsers too)
cc @nburt

Reply:
Ok I think your suggestion works. It seems like we have a UK bug but since we don't have different text for Canadians, Brits, or Americans it doesn't matter.